### PR TITLE
user story 7 done

### PR DESCRIPTION
**Pull request by sarynm12**
Shelter reviews user stories complete

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -33,7 +33,13 @@ def update
...
       redirect_to "/shelters/#{@shelter.id}"
     end
+  end
 
+  def destroy
+    @review = Review.find(params[:id])
+    @shelter = Shelter.find(@review.shelter_id)
+    @review.destroy
+    redirect_to "/shelters/#{@shelter.id}"
   end
```
Comment (lkriffell) [inline app/controllers/reviews_controller.rb:43]: Nice. I see our destroy method

File: spec/features/reviews/show_spec.rb [hunk 2]
```diff
@@ -43,6 +43,8 @@
 
       expect(current_path).to eq("/shelters/#{@shelter1.id}")
 
+      click_on "Delete Review"
+      expect(page).to_not have_content("Terrible Service")
```
Comment (lkriffell) [inline spec/features/reviews/show_spec.rb:48]: Nice test to see if the object has actually been deleted

File: app/views/shelters/show.html.erb [hunk 3]
```diff
@@ -19,12 +19,13 @@
...
       <%= image_tag "#{review.image}" %>
       <% end %>
-      <h2><%= link_to "Edit Review", "/shelters/#{@shelter.id}/#{review.id}/edit" %> </h2>
+      <h4><%= link_to "Edit Review", "/shelters/#{@shelter.id}/#{review.id}/edit" %> </h4>
+      <h4><%= link_to "Delete Review", "/shelters/#{@shelter.id}/#{review.id}", :method => :delete %> </h4>
     <% end %>
 
 
-    <h4><%= link_to "Add Review", "/shelters/#{@shelter.id}/new" %> </h4>
+    <h2><%= link_to "Add Review", "/shelters/#{@shelter.id}/new" %> </h2>
```
Comment (fieldstyler) [inline app/views/shelters/show.html.erb:31]: I like the new styling editions

File: coverage/.last_run.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 83.07
+    "covered_percent": 100.0
```
Comment (fieldstyler) [inline coverage/.last_run.json:3]: WOOT WOOT!!!!

File: app/views/shelters/show.html.erb (hunk 3 above)
Comment (lkriffell) [inline app/views/shelters/show.html.erb:31]: Nice. I like the formatting and our delete link is functional now

File: spec/features/reviews/show_spec.rb [hunk 5]
```diff
@@ -43,6 +43,8 @@
...
```
Comment (fieldstyler) [inline spec/features/reviews/show_spec.rb:47]: Nice test for deletions with 'to_not'

**Comment by fieldstyler** [comment]
Reviewed by Logan and Tyler

**Pull request merged**